program/c/src/oracle/for_cargo_test/cpyth_test.c

```diff
@@ -0,0 +1,23 @@
+/// The goal of this file is to provide the upd_aggregate function to local rust tests
+
+/// The heap address is different on solana runtime and locally, so here we need to allocate some heap space
+/// In compilation to bpf, PC_HEAP_START is provided by <solana.h>
+char heap_start[8192];
+#define PC_HEAP_START (heap_start)
+#define static_assert _Static_assert
```

Comment:
I copied this from `test_oracle.c` where they also have to deal with `bpf` vs local execution